mods/raclassic/rules/husks.yaml

```diff
@@ -81,6 +144,66 @@ MGG.Husk:
 	RenderSprites:
 		Image: mgg.destroyed
 
+MRJ.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Mobile Radar Jammer)
+	ThrowsParticle@spinner:
+		Anim: spinner
+	TransformOnCapture:
+		IntoActor: mrj
+	RenderSprites:
+		Image: mrj.destroyed
+
+TTNK.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Tesla Tank)
+	WithIdleOverlay@spinner:
+		Sequence: spinner
+	TransformOnCapture:
+		IntoActor: ttnk
+	RenderSprites:
+		Image: ttnk.destroyed
+
+DTRK.Husk
```

Comment:
DTRK husk sounds a bit unnecessary. It dies with the second blast already. I think the truck would be broken apart already with such a big explosion. Would like to hear hamb's opinion too tho.